libs/vkd3d/command.c

```diff
@@ -3268,7 +3272,7 @@ static void STDMETHODCALLTYPE d3d12_command_list_CopyBufferRegion(d3d12_command_
     d3d12_command_list_track_resource_usage(list, dst_resource);
     d3d12_command_list_track_resource_usage(list, src_resource);
 
-    d3d12_command_list_end_current_render_pass(list);
+    d3d12_command_list_end_current_render_pass(list, true);
```

Comment:
Is it possible to trigger CopyBufferToImage or ImageToBuffer here?

Reply:
CopyBufferRegion strictly maps to `vkCmdCopyBuffer`, and aliasing between image and buffer resources is undefined behaviour in D3D12 as far as I'm aware.